get

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+ARCH=${1:-amd64}
+VERSION=${2:-}
```

Comment:
Can we default to "latest", and fetch latest in that case (instead of the current `""`)? If so, it would be nice to have version first and arch in the end. You often might want a pinned version, but without explicitly setting the arch.

Reply:
Fixed that by the rework of the script, see https://github.com/cri-o/cri-o/pull/4552#discussion_r579089773